common/customizable_localization/00_ghw_tenet_custom_loc.txt

```diff
@@ -19,8 +19,11 @@ ghw_from_which_tenet = {
 
 	text = {
 		trigger = {
-			faith = { has_doctrine = tenet_warmonger }
+
+			# Warcraft
+			faith = { has_doctrine_tenet_warmonger_trigger = yes }
+
 		}
 		localization_key = ghw_from_which_tenet_warmonger
```

Comment:
> # GHWs given by which tenet?

> localization_key = ghw_from_which_tenet_warmonger

>  ghw_from_which_tenet_warmonger:0 "$tenet_warmonger_name$"

It will be shown that GHW is given by warmonger tenet, despite the fact that it is given by special doctrine,

Reply:
Done in 630576164d7f975ec1719315a94bacb554618712.